frontend/test/metabase/scenarios/admin/settings/settings.cy.spec.js

```diff
@@ -178,6 +178,27 @@ describe("scenarios > admin > settings", () => {
     cy.contains(/^February 11, 2019, 9:40 PM$/);
   });
 
+  it("should search for and select a new timezone", () => {
+    cy.server();
+    cy.route("PUT", "**/report-timezone").as("reportTimezone");
+
+    cy.visit("/admin/settings/localization");
+    cy.contains("Report Timezone")
+      .closest("li")
+      .find(".AdminSelect")
+      .click();
+
+    cy.focused().type("Centr");
+    cy.focused()
+      .closest(".MB-Select")
+      .find("a")
+      .last()
+      .click({ force: true });
```

Comment:
```suggestion
    cy.findByText("US/Central").click({ force: true });
```
We don't know if order is going to change, or if it depends on the context. Using the "last" link can backfire in the future.
This is much more explicit and it saves us a couple of lines :)